Leanplum/Leanplum.m

```diff
@@ -151,6 +157,49 @@ + (void)setNetworkTimeoutSeconds:(int)seconds forDownloads:(int)downloadSeconds
     [LPApiConstants sharedState].networkTimeoutSecondsForDownloads = downloadSeconds;
 }
 
++ (BOOL)isRichPushEnabled
+{
+    NSString *plugInsPath = [NSBundle mainBundle].builtInPlugInsPath;
+    NSDirectoryEnumerator *enumerator = [[NSFileManager defaultManager]
+                                         enumeratorAtPath:plugInsPath];
+    NSString *filePath;
+    while (filePath = [enumerator nextObject]) {
+        if([filePath hasSuffix:@".appex/Info.plist"]) {
+            NSString *newPath = [[plugInsPath stringByAppendingPathComponent:filePath]
+                                 stringByDeletingLastPathComponent];
+            NSBundle *currentBundle = [NSBundle bundleWithPath:newPath];
+            NSDictionary *extensionKey =
+            [currentBundle objectForInfoDictionaryKey:@"NSExtension"];
+            if ([[extensionKey objectForKey:@"NSExtensionPrincipalClass"]
+                 isEqualToString:@"NotificationService"]) {
+                return YES;
+            }
+        }
+    }
+    return NO;
+}
+
++ (NSString *)platform
```

Comment:
Should we move this to another class ?

Reply:
Yes please, even if we want these methods to exist, they should just wrap the other class

Reply:
yes, makes sense to make Leanpum.m light weight.

Reply:
done